bundle-workflow/src/test_workflow/test_publisher.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../src"))
```

Comment:
Is there any better way to do this? Require this to import aws module below

Reply:
This should not be required at all when run from the test.sh driver because it introduces a stable path. That's why we have those things.